Sources/ArgumentParser/Parsing/Interactive.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  File.swift
+//
+//
+//  Created by Keith  on 2022/6/21.
+//
+
+extension CommandParser {
+  func canInteract(error: Error, split: inout SplitArguments) -> Bool {
+    guard let error = error as? ParserError else { return false }
+    guard case let .missingValueForOption(inputOrigin, name) = error else { return false }
+    
+    var input: String?
+    while input?.isEmpty ?? true {
+      print("Please enter value for '\(name.synopsisString)': ", terminator: "")
+      input = readLine() ?? nil
+    }
+    
+    let inputIndex = inputOrigin.elements.first!.baseIndex! + 1
+    split._elements.insert(.init(value: .value(input!),
+                                 index: .init(inputIndex: .init(rawValue: inputIndex))),
+                           at: inputIndex)
+    
+    for index in (inputIndex + 1) ..< split.count {
+      split._elements[index].index = .init(inputIndex: .init(rawValue: index))
+    }
+    
+    split.originalInput.insert(input!, at: inputIndex)
```

Comment:
If we don't change `originalInput` here, even if we get the correct second input, `repeat --count --include-counter hello` will crash below:

https://github.com/apple/swift-argument-parser/blob/c61fc0f7f0b10fbbdbdc7e0b31cf1b7f8fd35ea0/Sources/ArgumentParser/Parsing/SplitArguments.swift#L271